src/Library/demos/Spin Button/main.js

```diff
@@ -0,0 +1,48 @@
+import Gtk from "gi://Gtk?version=4.0";
+import GObject from "gi://GObject";
+
+Gtk.init();
```

Comment:
```suggestion
import Gtk from "gi://Gtk";
import GObject from "gi://GObject";
```

unecessary in Workbench